kontakt-reservasjonsregister-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/krr/LookupParameters.java

```diff
@@ -0,0 +1,32 @@
+package no.difi.meldingsutveksling.serviceregistry.krr;
+
+public class LookupParameters {
+    private final String identifier;
+    private String clientOrgnr;
+
+    public LookupParameters(String identifier) {
+        this.identifier = identifier;
+    }
+
+    public String getIdentifier() {
+        return identifier;
+    }
+
+    public String getClientOrgnr() {
+        return clientOrgnr;
+    }
+
+    public void setClientOrgnr(String clientOrgnr) {
+        this.clientOrgnr = clientOrgnr;
+    }
+
+    public LookupParameters onBehalfOf(String clientOrgnr) {
+        setClientOrgnr(clientOrgnr);
+        return this;
+    }
+
+    public static LookupParameters lookup(String identifier) {
+        final LookupParameters lookupParameters = new LookupParameters(identifier);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "lookupParameters". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1488)